script/universal/MultisigBase.sol

```diff
@@ -28,8 +28,9 @@ abstract contract MultisigBase is Simulator {
         uint256 nonce = safe.nonce();
         console.log("Safe current nonce:", nonce);
 
-        if (bytes(vm.envOr("SAFE_NONCE", "")).length > 0) {
-            nonce = vm.envUint("SAFE_NONCE");
+        uint256 providedNonce = vm.envOr("SAFE_NONCE", nonce);
```

Comment:
alternative might just be `nonce = vm.envOr("SAFE_NONCE", nonce);` but this works

Reply:
yeah only thing was doesnt really make sense if the nonce is lower than the current nonce, so in that case we defualt to the current nonce here. but I could actually see someone maybe wanting to do that for testing (simulating some tx from the past)